mds/apis/agency_api/v0_x/vehicles.py

```diff
@@ -102,8 +106,12 @@ class DeviceTelemetrySerializer(serializers.Serializer):
 
 
 class DeviceEventSerializer(serializers.Serializer):
+    event_time = serializers.DateTimeField(help_text="Event time")
     event_type = serializers.ChoiceField(
-        choices=enums.EVENT_TYPE_CHOICES, help_text="Vehicle event."
+        choices=enums.EVENT_TYPE_CHOICES, help_text="Vehicle event type."
+    )
+    event_type_reason = serializers.ChoiceField(
```

Comment:
which version of MDS spec are you using for these change ? did you consider ongoing PRs ?

Reply:
Just following the currently published provider spec. No ongoing PR is changing how an event type or event type reason is defined.

I'll rework this PR not to change `device_status`, and `event_type_reason` will match our `event_type` field.

Reply:
This is the Agency API - it should not follow the provider API spec, but the agency spec - or am I missing something ?